log/report-hera.intel-log/rtreport.txt

```diff
@@ -3493,14 +3370,80 @@ TEST #47: PASS
   mediator_FBOcn_o_restart.nc: bit-for-bit identical
   mediator_scalars_restart.txt: bit-for-bit identical
   Executable did not change during test suite:
-    File: /scratch1/BMC/gmtb/Dom.Heinzeller/FV3_RT/rtgen.193663/exec/fv3_mom6_cice.exe
-    Expected md5sum: 881a58784947f0f0cfe53ef2e977ba8b
-    Actual md5sum: 881a58784947f0f0cfe53ef2e977ba8b
-  TEST PASSED AT Mon Mar 30 19:02:07 UTC 2020
-
-TEST #48: PASS
-  Test cpld_fv3_ccpp_384_mom6_cice_ww3_cold_bmark_rt_repro starting at Mon Mar 30 20:36:44 UTC 2020 (Coupled FV3-CCPP-MOM6-CICE-WW3 system - cold 384 - repro)
-  Mon Mar 30 20:36:45 UTC 2020
+    File: /scratch1/NCEPDEV/stmp2/Bin.Li/scrub/rtgen.2781/exec/fv3_mom6_cice.exe
+    Expected md5sum: 82010132a6265a40fb6527c77ea925e6
+    Actual md5sum: 82010132a6265a40fb6527c77ea925e6
+  TEST PASSED AT Sun Apr 12 21:08:21 UTC 2020
+  FAIL: job state is DEAD
+
+TEST #48: FAIL
+  Test cpld_fv3_ccpp_384_mom6_cice_ww3_cold_bmark_rt_repro starting at Sun Apr 12 23:21:19 UTC 2020 (Coupled FV3-CCPP-MOM6-CICE-WW3 system - cold 384 - repro)
+  Sun Apr 12 23:21:19 UTC 2020
+WORKFLOW REPORT AT Sun Apr 12 23:54:22 2020 (+1586735662)
+Tests: 3 failed, 45 passed out of 48
```

Comment:
3 tests failed?

Reply:
These 3 failed tests have been rerun until they are successful.

Reply:
Good catch!

Reply:
> These 3 failed tests have been rerun until they are successful.

So the 3 failed tests have run and all passed?

Reply:
Yes, they have been run successfully before this pull request.

Reply:
Ok, thanks. I did not understand the earlier comment. I will complete the review now.